bootstrap.php

```diff
@@ -14,4 +14,19 @@
  *
  */
 
-require_once __DIR__ . '/vendor/autoload.php';
+$vendorDir = __DIR__ . '/vendor';
+$importedVendorDir = __DIR__ . '/../../../vendor';
+
+/**
+ * Check whether this SDK got imported via composer. Composer imports required libraries to
+ * vendor/<vendor_namespace>/<project_namespace>
+ */
+if (is_dir($importedVendorDir)) {
```

Comment:
I think we should do this the other way round: only fallback if `$vendorDir` doesn't exist. If it does, all is good.

If it deosn't, then we output a warning saying that the bootstrap should not be used, and try the `$importedVendorDir` (which, also, may or may not exist).

I can think of weird scenarios where a user would have manually created a `vendor` dir, and unpacked the dist zip into it.